src/test/java/nextstep/courses/domain/CourseTest.java

```diff
@@ -0,0 +1,20 @@
+package nextstep.courses.domain;
+
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class CourseTest {
+
+    @Test
+    @DisplayName("과정(Course)은 기수 단위로 운영하며, 여러 개의 강의(Session)를 가질 수 있다.")
+    void canHasManySessions() {
+        Course course = new Course("객체지향 프로그래밍", 1L);
+        assertThat(course)
```

Comment:
```suggestion
        course.contains(session);
```

`여러 개의 강의(Session)를 가질 수 있다.`
객체의 public 메소드를 테스트 해보는것은 어떨까요?
물론 테스트를 위한 메소드이지만, 단순히 `sessions`라는 이름을 가진 상태를 가진다고 `여러개`라는 의미를 지키는지에 대한 최소한의 테스트가 될까요?